contrib/cirrus/lib.sh

```diff
@@ -41,7 +41,7 @@ fi
 
 OS_RELEASE_ID="$(source /etc/os-release; echo $ID)"
 # GCE image-name compatible string representation of distribution _major_ version
-OS_RELEASE_VER="$(source /etc/os-release; echo $VERSION_ID | cut -d '.' -f 1)"
+OS_RELEASE_VER="$(source /etc/os-release; echo $VERSION_ID | tr -d '.')"
```

Comment:
Note: This is needed to support *two* major-version `20` Ubuntu VMs.  For example, previously `$OS_RELEASE_VER` was `20`, now it's `2010`.  This enables simple version comparisons in scripts, since bash does not like floating-point numbers.